recipe/migrations/cuda110.yaml

```diff
@@ -31,14 +31,22 @@ __migrator:
       - quay.io/condaforge/linux-anvil-cos7-cuda:10.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:10.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cos7-cuda:10.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-ppc64le-cuda:10.2  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+      - quay.io/condaforge/linux-anvil-aarch64-cuda:10.2  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
```

Comment:
We do not support `CUDA 10.2` for `aarch64`

Reply:
Ah right. Was look at `ppc64le`

Yeah starting with 11.0 makes sense. Maybe we can just do that with both. Thoughts @jaimergp?

We've been thinking about dropping the CUDA versions pre-11.0 anyway ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 )